eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -88,13 +88,17 @@ function GetRelativePath($path) {
 }
 
 $allPackageProperties = Get-AllPkgProperties $serviceDirectory
-if ($allPackageProperties)
+
+# Only export package info JSON files for track 2 packages
+$targetPackages = $allPackageProperties.Where({ $_.IsNewSdk })
```

Comment:
Instead of filtering it might be safer to sort on this field and don't allow overwriting below if they file already exists. I say it might be safer as these files might be interesting for doing a release of a track 1 package outside of the docs context.

Reply:
This can be accomplished but the check is not as simple... `SetOutput` will read in properties from an existing file in the case of `-addDevVersion` so we'd have to keep track of files that have been written to in the execution of the script and avoid overwriting that way.

Reply:
Fair I forgot that we do overwrite for that case. Ok Lets go ahead and figure out track 1 and if we find that that causes problems we can re-evaluate.